### ISSUE TITLE
Flyout: hide "current version" for single version projects

### ISSUE THREAD

**Issue by humitos** [readthedocs/addons#97]
Currently we are showing the `v: <default version>` (e.g. `v: latest`) in the flyout when it's closed for "single version" projects. Now that we have the Read the Docs logo on it, we should probably remove the `v: latest` from here and only show the logo when it's a single version project.

![Screenshot_2023-08-30_12-13-24](https://github.com/readthedocs/addons/assets/244656/00b033ca-601c-4b0f-af99-206de7be6296)

**Comment by agjohnson** [comment]
Also, it might be good to consider what we might want to change with the flyout before we optimize what we have there now. We've started with the current flyout implementation, but we've also thrown around a lot of ideas about what we might put in the collapsed/expanded versions of the flyout menu if we were designing it today.

**Comment by humitos** [comment]
Yeah, this issue may be a non-issue for the current flyout and something to consider in the redesign of it.

I'm not sure what we did in the old version. Maybe we just need to copy the logic from there if it's different.
